olp-cpp-sdk-dataservice-read/include/olp/dataservice/read/VersionedLayerClient.h

```diff
@@ -96,8 +98,42 @@ class DATASERVICE_READ_API VersionedLayerClient final {
    * @param layer_id The layer ID of the versioned layer from which you want to
    * get data.
    * @param settings The `OlpClientSettings` instance.
+   *
+   * @deprecated Will be removed by 06.2020.
+   */
+  OLP_SDK_DEPRECATED(
+      "Use the ctor with the explicitly specified version. This ctor is "
+      "deprecated and will be removed "
+      "by 06.2020")
```

Comment:
Can we change it to:
```suggestion
      "Deprecated and will be removed "
      "by 06.2020"
      "Use the ctor with the explicitly specified version instead")
```

Reply:
Please also re-format the string to two lines.